src/core/ActionRegister.h

```diff
@@ -29,7 +29,7 @@
 #include "tools/Keywords.h"
 #include <memory>
 
-namespace PLMD {
+namespace PLMD { //{
```

Comment:
@Iximiel why this additional open brace after the comment?

Reply:
I'm not a vim user, but is what I understood to do from [this](https://github.com/plumed/plumed2/pull/934#discussion_r1190056226)